packages/common/src/core/schemas/internal/getThumbnailUiSchemaElement.ts

```diff
@@ -29,13 +43,18 @@ export function getThumbnailUiSchemaElement(
       alwaysShow: true,
     });
   }
+  const defaultEntityThumbnail =
+    DEFAULT_ENTITY_THUMBNAILS[entityType] ?? DEFAULT_ENTITY_THUMBNAILS.content;
+  const defaultImgUrl = getCdnAssetUrl(defaultEntityThumbnail, requestOptions);
+
   return {
     labelKey: "shared.fields._thumbnail.label",
     scope: "/properties/_thumbnail",
     type: "Control",
     options: {
       control: "hub-field-input-image-picker",
       imgSrc: thumbnailUrl,
+      defaultImgSrc: defaultImgUrl,
```

Comment:
Apologies, I may not have been clear in my last round of feedback. I would also expect the property to be named `defaultImgUrl` on the `options` object here. I.e. all occurrences of `defaultImgSrc` => `defaultImgUrl` on this PR.